docs/topics/RPC.md

```diff
@@ -413,9 +413,9 @@ Used to get a channel the client is in.
 | name | String | channel name |
 | type | Number | channel type (guild text: 0, guild voice: 2, dm: 1, group dm: 3) |
 | topic | String | (text) channel topic |
-| bitrate | Int | (voice) bitrate of voice channel |
-| user_limit | Int | (voice) user limit of voice channel (0 for none) |
-| position | Int | position of channel in channel list |
+| bitrate | integer | (voice) bitrate of voice channel |
```

Comment:
I've decided to undo all the changes in RPC.md. There's a lot that needs to be worked on there, so I'll do a separate commit with the fixes for that page